packages/terra-table/src/Table.jsx

```diff
@@ -141,7 +146,13 @@ const propTypes = {
    * Boolean indicating whether or not the table should allow entire rows to be selectable. An additional column will be
    * rendered to allow for row selection to occur.
    */
-  hasSelectableRows: PropTypes.bool,
+  // hasSelectableRows: PropTypes.bool,
```

Comment:
Since we are removing this prop as part of a MVB, we can probably remove this from propTypes entirely

Reply:
I missed removing this commented logic.  I will do that.  However, there will not be a MVP since we have not yet done the 1.0 release.